### PR TITLE
Update: Annapurna HardFork

### PR DESCRIPTION
**Pull request by smin-k**

### CODE REVIEW

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -1104,8 +1095,6 @@ func setBootstrapNodes(ctx *cli.Context, cfg *p2p.Config) {
 		urls = params.GoerliBootnodes
 	case ctx.Bool(KilnFlag.Name):
 		urls = params.KilnBootnodes*/
-	case ctx.Bool(LveFlag.Name):
-		urls = params.LveBootnodes
 	case ctx.Bool(SeoulFlag.Name):
 		urls = params.SeoulBootnodes
 	case ctx.Bool(GwangjuFlag.Name):
```
Comment (siddharth0a) [inline cmd/utils/flags.go:1100]: Annapurna flag?

File: consensus/eccpow/LDPCDifficulty.go [hunk 2]
```diff
@@ -190,9 +190,9 @@ func MakeLDPCDifficultyCalculatorAnnapurna() func(time uint64, parent *types.Hea
 			//fmt.Printf("Uncle block exists")
 			x.Sub(big2, x)
 		}
-		//fmt.Printf("(2 if len(parent_uncles) else 1) - (block_timestamp - parent_timestamp) / 7 : %v\n", x)
+		//fmt.Printf("(2 if len(parent_uncles) else 1) - (block_timestamp - parent_timestamp) // 7 : %v\n", x)
 
-		// max((2 if len(parent_uncles) else 1) - (block_timestamp - parent_timestamp) // 9, -99)
+		// max((2 if len(parent_uncles) else 1) - (block_timestamp - parent_timestamp) // 7, -99)
```
Comment (siddharth0a) [inline consensus/eccpow/LDPCDifficulty.go:195]: we don't need these comment. Isn't it?

File: consensus/eccpow/consensus.go [hunk 3]
```diff
@@ -371,6 +373,12 @@ func calcDifficultySeoul(chain consensus.ChainHeaderReader, time uint64, parent
 	return difficultyCalculator(time, parent)
 }
 
+func calcDifficultyAnnapurna(chain consensus.ChainHeaderReader, time uint64, parent *types.Header) *big.Int {
+	difficultyCalculator := MakeLDPCDifficultyCalculatorAnnapurna()
+	//return difficultyCalculator(chain, time, parent)
+	return difficultyCalculator(time, parent)
```
Comment (siddharth0a) [inline consensus/eccpow/consensus.go:379]: These comments

File: cmd/utils/flags.go (hunk 1 above)
Comment (smin-k) [reply cmd/utils/flags.go:1100]: The flag refers to the name of the network, so it is not needed.

**Pull request merged**